services/src/main/java/io/druid/cli/CliBroker.java

```diff
@@ -105,6 +107,8 @@ public void configure(Binder binder)
             Jerseys.addResource(binder, ClientInfoResource.class);
             LifecycleModule.register(binder, QueryResource.class);
             LifecycleModule.register(binder, DruidBroker.class);
+            binder.bind(LookupReferencesManager.class).in(ManageLifecycle.class);
+            LifecycleModule.register(binder, LookupReferencesManager.class);
```

Comment:
@himanshug after testing via my IDE you are actually right, i hope @cheddar can provide a clarification ...